src/jit/ByteCodeParser.cpp

```diff
@@ -1741,7 +1745,7 @@ static void compileFunction(JITCompiler* compiler)
             info = Instruction::kIsCallback;
             compiler->increaseStackTmpSize(32);
 #endif /* SLJIT_CONFIG_ARM_32 */
-            requiredInit = OTMinMaxV128;
+            requiredInit = OTRMinMaxV128;
```

Comment:
I don't think this is correct. Move All reaxed to another group and keep the original.